ckanext/data_qld/google_analytics/plugin.py

```diff
@@ -37,7 +37,9 @@ def run(self):
             # User-Agent must be present
             # GA might ignore a custom UA so fall back to imitating Firefox
             headers['User-Agent'] = data_dict.pop('user_agent', 'Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:47.0) Gecko/20100101 Firefox/47.0')
-            log.debug("Sending API event to Google Analytics: %s", data_dict['events'][0]['params']['action'])
+            log.info("Sending API event to Google Analytics: %s", data_dict['events'][-1]['params']['action'])
```

Comment:
ops works needs to have 
logger
ckanext.googleanalytics set to info so trace/debug info is not emitted.